modules/dkan/dkan_dataset/dkan_dataset.forms.inc

```diff
@@ -342,9 +342,28 @@ function dkan_dataset_form_alter(&$form, &$form_state, $form_id) {
     $form['field_link_api'][$field_link_api_langcode][0]['#title'] = '';
     $form['field_link_remote_file'][$field_link_remote_file_langcode][0]['#title'] = '';
 
-    // Resources inherit groups from parent dataset.
-    // Hide field so users cannot manually modify that information.
-    $form['og_group_ref']['#access'] = FALSE;
+    $groups = dkan_dataset_get_group_options();
```

Comment:
It looks to me like this is not working correctly. It appears to now shows the group field on all resources

Reply:
That line being removed is OK. On line 355 we are hiding or showing the field using #states so that's the reason for this change here.